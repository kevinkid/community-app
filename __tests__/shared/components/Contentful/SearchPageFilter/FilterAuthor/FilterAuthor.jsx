import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import { FilterAuthorInner } from 'components/Contentful/SearchPageFilter/FilterAuthor';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    theme: {},
    selected: 'All authors',
  };
  renderer.render(<FilterAuthorInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
