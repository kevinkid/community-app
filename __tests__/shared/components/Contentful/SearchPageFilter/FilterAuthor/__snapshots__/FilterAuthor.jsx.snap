// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div
  className="undefined "
>
  <span>
    Author
  </span>
  <button
    onClick={[Function]}
    type="button"
  >
    <span>
      All authors
    </span>
    <IconArrow
      height="10"
      viewBox="0 0 16 10"
      width="16"
      xmlns="http://www.w3.org/2000/svg"
    />
  </button>
</div>
`;
